daemon/stores/base.py

```diff
@@ -92,6 +95,7 @@ def __delitem__(self, key: DaemonID) -> None:
         .. #noqa: DAR201"""
         self.status.items.pop(key)
         self.status.num_del += 1
+        self.status.size -= 1
```

Comment:
`size` should always be handled in the pydantic model using the validator. If it is not correctly updated, fix should go in `StoreStatus`

Reply:
Now that I think of it, `size` is just the length of items, it can just be `len(store)`. I guess, we can remove size?